### PR TITLE
[Console] Escape default value when dumping help

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by ogizanagi** [comment]
The same will apply to arguments (fixed by this patch, but no there is no fixtures covering it).

**Comment by lyrixx** [comment]
> The same will apply to arguments (fixed by this patch, but no there is no fixture covering it).

I know. Adding test is a bit boring + it's already fixed. That's why I did not added fixtures for that.

File: src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php [hunk 1]
```diff
@@ -43,6 +43,8 @@ public static function getInputOptions()
             'input_option_4' => new InputOption('option_name', 'o', InputOption::VALUE_IS_ARRAY | InputOption::VALUE_OPTIONAL, 'option description', array()),
             'input_option_5' => new InputOption('option_name', 'o', InputOption::VALUE_REQUIRED, "multiline\noption description"),
             'input_option_6' => new InputOption('option_name', array('o', 'O'), InputOption::VALUE_REQUIRED, 'option with multiple shortcuts'),
+            'input_option_7' => new InputOption('option_name', 'o', InputOption::VALUE_REQUIRED, 'option description', '<comment>style</>'),
```
Comment (xabbuh) [inline src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php]: We should IMO use more descriptive names. Adding a counter suffix leads to merge conflicts that could be avoided.

Comment (lyrixx) [inline src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php]: Indeed. But I don't know if I should do that right now, in this PR?!

Comment (ogizanagi) [inline src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php]: Agreed. Also it should be scoped to targeted descriptors IMHO (as in #20900).

Comment (fabpot) [inline src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php]: I would indeed  the **new** ones better.

Comment (lyrixx) [inline src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php]: I'm not sure to understand you. So I changed the 'key' and I also added tests for the arguments.

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Comment by nicolas-grekas** [comment]
Thank you @lyrixx.

**Pull request merged**